<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDoubleSpinBox" name="doubleSpinBox_longitude">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>108.950000000000003</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_latitude">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>34.869999999999997</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_height">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>697.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>经度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>高程</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>距离</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_r">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-9999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
    <property name="value">
     <double>2285907.129999999888241</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>俯仰</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>方位</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_a">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>33.334600000000002</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_e">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-90.000000000000000</double>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
    <property name="value">
     <double>4.105800000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>测站属性</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>测站测量值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>地心系测量值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-99999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_z">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>8</number>
    </property>
    <property name="minimum">
     <double>-9999999999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ToXYZ">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PAE to XYZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ToPAE">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>XYZ to PAE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>921</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>2018 12 07 02 41 43.572     -2138671.149     -3916864.311      5283931.870    -4082.624797    -4234.640896    -4791.027825
2018 12 07 10 41   43.00000         2285907.13   33.3346    4.1058</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
